Formula/volta.rb

```diff
@@ -0,0 +1,19 @@
+class Volta < Formula
+  desc "JavaScript toolchain manager for reproducible environments"
+  homepage "https://volta.sh"
+  url "https://github.com/volta-cli/volta.git",
+    :revision => "10a9bee148d2a833069d6cc27aea56af3092dc6e",
+    :tag      => "v0.7.2"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root", prefix, "--path", "."
```

Comment:
Could we perhaps do:

```suggestion
    system "cargo", "install", "--locked", "--root", prefix, "--path", "."
```